### PR TITLE
Fix: use the email input type where needed

### PR DESCRIPTION
**Pull request by ariarijp**
I think, use the email input type for a good UX.

### CODE REVIEW

File: rd_ui/app/login.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
   <form role="form" method="post" name="login">
     <div class="form-group">
       <label for="inputEmail">Email</label>
-      <input type="text" class="form-control" id="inputEmail" name="email" value="{{email}}">
+      <input type="email" class="form-control" id="inputEmail" name="email" value="{{email}}">
```
Comment (arikfr) [inline rd_ui/app/login.html]: While it makes sense, it's a problem as sometimes "email" is used just as a username -- like the default `admin` user. Changing type here will prevent people from logging in...

File: redash/templates/forgot.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 <form role="form" method="post">
     <div class="form-group ">
         <label for="email">Enter the email address you used for this account:</label>
-        <input type="text" class="form-control" name="email" value="{{email}}">
+        <input type="email" class="form-control" name="email" value="{{email}}">
```
Comment (arikfr) [inline redash/templates/forgot.html]: Here we can keep the email type, as it makes no sense to use the "forgot password" page for non emails.

My suggestion: update the pull request to change the type only here.

**Comment by arikfr** [comment]
Thanks! See my comments.

File: redash/templates/forgot.html (hunk 2 above)
Comment (ariarijp) [inline redash/templates/forgot.html]: Thank you for your review! I will do.

**Comment by ariarijp** [comment]
Updated this pull request.

Please review it again.

**Comment by arikfr** [comment]
Thanks!

**Pull request merged**